.travis.yml

```diff
@@ -31,9 +31,9 @@ before_install:
   - conda update -y -q conda
   - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CONDA_COMPILERS="gcc_linux-64 gfortran_linux-64"; fi
   - if [[ $TRAVIS_OS_NAME == "osx" ]]; then export CONDA_COMPILERS="clang_osx-64 gfortran_osx-64"; fi
-  - if [[ $CMOR_PYTHON_VERSION == '2.7' ]]; then conda create -q -n py2.7 -c cdat/label/nightly -c conda-forge -c cdat ossuuid udunits2 hdf5 libnetcdf numpy openssl lazy-object-proxy cdms2 python=2.7 $CONDA_COMPILERS testsrunner; fi
-  - if [[ $CMOR_PYTHON_VERSION == '3.6' ]]; then conda create -q -n py3.6 -c cdat/label/nightly -c conda-forge -c cdat ossuuid udunits2 hdf5 libnetcdf numpy openssl lazy-object-proxy cdms2 python=3.6 $CONDA_COMPILERS testsrunner; fi
-  - if [[ $CMOR_PYTHON_VERSION == '3.7' ]]; then conda create -q -n py3.7 -c cdat/label/nightly -c conda-forge -c cdat ossuuid udunits2 hdf5 libnetcdf numpy openssl lazy-object-proxy cdms2 python=3.7 $CONDA_COMPILERS testsrunner; fi
+  - if [[ $CMOR_PYTHON_VERSION == '2.7' ]]; then conda create -q -n py2.7 -c cdat/label/nightly -c conda-forge -c cdat libuuid udunits2 hdf5 libnetcdf numpy openssl lazy-object-proxy cdms2 python=2.7 $CONDA_COMPILERS testsrunner; fi
```

Comment:
couldn't we replace all these lines with:
```
conda create -q -n py$CMOR_PYTHON_VERSION -c cdat/label/nightly -c conda-forge -c cdat libuuid udunits2 hdf5 libnetcdf numpy openssl lazy-object-proxy cdms2 python=$CMOR_PYTHON_VERSION $CONDA_COMPILERS testsrunner
```

Reply:
Using $CMOR_PYTHON_VERSION in the command doesn't work.  I will try to fix that in another pull request.